centaur/src/main/resources/standardTestCases/wdl_draft3/draft3_http_imports/draft3_http_imports.wdl

```diff
@@ -0,0 +1,11 @@
+version 1.0
+
+import "https://raw.githubusercontent.com/broadinstitute/cromwell/develop/centaur/src/main/resources/standardTestCases/wdl_draft3/draft3_infer_version/draft3_infer_version.wdl" as infered
```

Comment:
It would be nice to do this with a mock server instead of github for stability, but not a blocker by any means

Reply:
Don't think this is a blocker either but `develop` is also a floating tag. Links to github should [use a permalink](https://help.github.com/articles/getting-permanent-links-to-files/).